plugins/modules/mysql_info.py

```diff
@@ -121,7 +122,7 @@
   description: Database server version.
   returned: if not excluded by filter
   type: dict
-  sample: { "version": { "major": 5, "minor": 5, "release": 60 } }
+  sample: { "version": { "major": 5, "minor": 5, "release": 60, "suffix": "MariaDB", "full_version": "5.5.60-MariaDB" } }
```

Comment:
```suggestion
  sample: { "version": { "major": 5, "minor": 5, "release": 60, "suffix": "MariaDB", "full": "5.5.60-MariaDB" } }
```
@rndmh3ro maybe just `full`? IMO its invocation `version.full` in playbooks would look better than `version.full_version`.
what do you think?

Reply:
@rndmh3ro BTW if you like to contribute more, you could add `full` (but w/o `suffix` IMO) to https://github.com/ansible-collections/community.postgresql/blob/main/plugins/modules/postgresql_info.py and https://github.com/ansible-collections/community.postgresql/blob/main/plugins/modules/postgresql_ping.py 
What do you think? (Or let me know if you're busy, i could do it)